Content/ContentTypeResolver/SnippetSelectionResolver.php

```diff
@@ -79,18 +70,13 @@ public function resolve($data, PropertyInterface $property, string $locale, arra
 
         $snippets = [];
```

Comment:
would be nice to use the `DefaultSnippetManager::loadIdentifier()` method in this class too